src/use_case/create_meeting/CreateMeetingGmailDataAccessInterface.java

```diff
@@ -0,0 +1,11 @@
+package use_case.create_meeting;
+
+import com.google.api.services.gmail.model.Message;
+
+import javax.mail.MessagingException;
+import java.io.IOException;
+
+public interface CreateMeetingGmailDataAccessInterface {
+    Message sendMeetingCreationEmail(String email, String meetingName) throws MessagingException, IOException;
```

Comment:
need extra parameter of String toEmail (emails that you are sending to)

Reply:
For this:
I believe String email is a string that can be converted to an ArrayList of emails.
I thought I could make the first element in this list the sender and the every element (including the first) the recipients. 

But honestly it might be easier to just require the parameters fromEmail and toEmail.

Reply:
What do you think?

Reply:
Okay never mind let me think this through again.

Reply:
It's up to you, but the GmailDAO needs toEmail, fromEmail, titleText, and bodyText. So as long as you can get those working, it should be fine.